lib/features/create/Create.js

```diff
@@ -1,130 +1,115 @@
-var LOW_PRIORITY = 750;
-
 var MARKER_OK = 'drop-ok',
     MARKER_NOT_OK = 'drop-not-ok',
     MARKER_ATTACH = 'attach-ok',
     MARKER_NEW_PARENT = 'new-parent';
 
 import {
-  append as svgAppend,
-  attr as svgAttr,
-  classes as svgClasses,
-  create as svgCreate,
-  remove as svgRemove
-} from 'tiny-svg';
-
-import { assign } from 'min-dash';
-
-import {
-  translate
-} from '../../util/SvgTransformUtil';
+  assign,
+  filter,
+  find,
+  forEach,
+  isArray,
+  isNumber,
+  map
+} from 'min-dash';
 
-import { Shape } from '../../model';
+import { getBBox } from '../../util/Elements';
 
 
 /**
- * Adds the ability to create new shapes via drag and drop.
```

Comment:
Will there be a replacement for the deleted documentation?

Reply:
Added documentation throughout the code.